.github/workflows/refresh-doc.yml

```diff
@@ -0,0 +1,75 @@
+name: refresh-doc
+on:
+  push:
+    branches:
+      - hannah-github-actions
+
+env:
+  WEB_URL: 'https://cold-pigs-fail-174-94-59-90.loca.lt' # This needs to be updated to the live web server!
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v3
+      
+    - name: Changed Files
+      id: changed-files
+      uses: tj-actions/changed-files@v18.7
+      with: # Any file in the docs
+        files: |
+          docs/**
+    
+    # --- Log all diff info ---
+    - name: Log the files commit SHA if any md files have diffs
```

Comment:
Is this debug code?

Reply:
I added all the logging for viewing purposes! I think when Mackenzie comes back we might change the payload to be more descriptive (modified vs added vs deleted), so I just logged them for now!